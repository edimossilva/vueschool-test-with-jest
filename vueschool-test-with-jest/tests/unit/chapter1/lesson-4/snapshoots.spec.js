const user = {
  name: 'Edimo Sousa Silva',
  age: 28,
  job: 'Developer'
};

test('user matches',() => {
  expect(user).toMatchSnapshot();
});