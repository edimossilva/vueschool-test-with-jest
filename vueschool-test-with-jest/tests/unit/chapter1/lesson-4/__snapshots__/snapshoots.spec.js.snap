// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
Object {
  "age": 28,
  "job": "Developer",
  "name": "Edimo Sousa Silva",
}
`;
